### PR TITLE
Support Julia 1.0

### PR DESCRIPTION
**Pull request by roberthoenig**
Also, remove support for Julia < 0.7

@oxinabox FYI.

### CODE REVIEW

**Comment by oxinabox** [comment]
@Evizero  I'll code review this now, I am sitting next to @roberthoenig

**Comment by Evizero** [comment]
Cool! but please use `refactor-0.6` as a starting point. A lot has changed there

**Comment by oxinabox** [comment]
I am sorry @roberthoenig ,
I thought there was another branch that was up to date, but I thought it was called `dev`.
And when I didn't see `dev` I thought it had finally been merged into `master`.

File: test/runtests.jl [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
-if VERSION >= v"0.5-"
-    using Base.Test
-else
-    using BaseTestNext
-    const Test = BaseTestNext
-end
+using Test
 
 tests = [
-    "classification.jl",
```
Comment (JoshChristie) [inline test/runtests.jl:12]: Commented out?

Comment (roberthoenig) [reply test/runtests.jl:12]: Whoops, yeah, I did some testing and forgot to undo that.

**Comment by roberthoenig** [comment]
> Cool! but please use release-0.6 as a starting point. A lot has changed there

Okay.  @Evizero for the future, I think `master` should always contain the latest changes? Or dev, but not any release-sounding branch like "refactor-0.6".

**Comment by Evizero** [comment]
You are of course right. To my (rather poor) defense, I though I was the only one using the package in its currently unfinished state. I am also experimenting with ideas on that branch that change over time. Such as the ROC stuff. As the name of the branch suggests it wasn't ever intended to live to see the 0.7 release

Anyway. sorry for the confusion.

**Comment by roberthoenig** [comment]
Closing this. I also looked into upgrading the newly updated master, without success: It depends on ReferenceTests, which in turn has a couple dependencies, many of which need to be upgraded to 1.0...

I think the best thing to do here is to wait for FemtoCleaner to do the work.

**Pull request closed without merging**